cranelift-codegen/src/ir/condcodes.rs

```diff
@@ -101,6 +109,8 @@ impl Display for IntCC {
             UnsignedGreaterThanOrEqual => "uge",
             UnsignedLessThan => "ult",
             UnsignedLessThanOrEqual => "ule",
+            Carry => "cr",
+            NotCarry => "nc",
```

Comment:
```suggestion
            NotCarry => "ncr",
```

Reply:
lol you're quick. I'll do a repo-wide search and replace.

Reply:
Done!